misc/ballerina-bindgen/src/main/java/org/ballerinalang/bindgen/utils/BindgenEnv.java

```diff
@@ -128,7 +128,7 @@ String getAliasClassName(String alias) {
         return aliases.get(alias);
     }
 
-    void setAlias(String alias, String className) {
+    public void setAlias(String alias, String className) {
```

Comment:
@IrushiL Now since its public shall we include method comments?

Reply:
Done